donkeycarManagerFront/src/js/service.js

```diff
@@ -22,7 +22,7 @@ import axios from 'axios'
 
 export default class DonkeycarManagerService {
   static get ip() {
-    return 'localhost'
+    return '0.0.0.0'
```

Comment:
should be 192.168.20.42

Reply:
but if someone run our manager it will be localhost

Reply:
I'm saying the ip should be 192.168.20.42, as it is the one that the computer running the backend will have, localhost was here for dev, and 0.0.0.0 is just never valid.